reference/forms/types/datetime.rst

```diff
@@ -72,6 +74,23 @@ date_widget
 
 .. include:: /reference/forms/types/options/date_widget_description.rst.inc
 
+date_label
+~~~~~~~~~~
+
+**type**: ``string`` | ``null`` **default**: The label is "guessed" from the field name
+
+.. versionadded:: 4.2
+    The ``date_label`` option was introduced in Symfony 4.2.
+
+Sets the label that will be used when rendering the date widget separately.
```

Comment:
I would remove the word "separately" here. It makes me understand the sentence as if the `date_label` option would only be used under certain circumstances.